lib/manageiq/api/common/middleware/web_server_metrics.rb

```diff
@@ -0,0 +1,53 @@
+module ManageIQ
+  module API
+    module Common
+      module Middleware
+        class WebServerMetrics
+          def initialize(app, options = {})
+            @app            = app
+            @metrics_prefix = options[:metrics_prefix] || "http_server"
+
+            require 'prometheus_exporter/client'
+            require 'prometheus_exporter/metric'
+
+            @counter   = PrometheusExporter::Client.default.register(:counter, "#{@metrics_prefix}_total", "The total number of HTTP requests handled by the Rack application.")
```

Comment:
I think this should be called something like `*_requests_total`. It feels like it's missing the name of the actual metrics it's measuring.

Reply:
We also may have more than one metric that is a counter so I think this should have a more specific variable name. Same for all the others below.